### ISSUE TITLE
non-default qualityspec results in process error

### ISSUE THREAD

**Issue by baadc0de** [lrbalt/libsoxr-rs#2]
When the `Soxr::new` constructor is given a `None` option for the `quality_spec` argument, the process code finishes as expected.

When specifying a custom `QualitySpec` like:

```rust
Some(QualitySpec::new(
  &QualityRecipe::VeryHigh,
  QualityFlags::HI_PREC_CLOCK,
))
```

The process method fails with "null pointer" error. As soon as the `quality_spec` is changed back to `None`, this error goes away.

I tried with other quality recipes and flags (including 0) with the same result.

Thank you for the awesome library!

**Comment by lrbalt** [comment]
I can reproduce the SIGSEGV. I'll investigate.

**Comment by lrbalt** [comment]
should be fixed now in 0.1.5
